python/servo/build_commands.py

```diff
@@ -324,6 +324,20 @@ def build(self, target=None, release=False, dev=False, jobs=None,
                     for ssl_lib in ["libcryptoMD.dll", "libsslMD.dll"]:
                         shutil.copy(path.join(env['OPENSSL_LIB_DIR'], "../bin" + msvc_x64, ssl_lib),
                                     servo_exe_dir)
+                    if os.environ.get("VS140COMNTOOLS", ""):
+                        vs_dir = os.environ["VS140COMNTOOLS"]
+                        vs_dll_dir = path.join(vs_dir, "..", "IDE", "Remote Debugger", os.environ["PLATFORM"].lower())
+                        os.environ['PATH'] = os.environ['PATH'] + os.pathsep + vs_dll_dir
```

Comment:
I don't like modifying environment variables in process, it can be confusing and potentially can cause memory leaks, etc. I believe you had an approach that involved copying before - can we use that instead? Or maybe symlinking?